manifests/c/CoreFTP/FTPServer/2.0-749/CoreFTP.FTPServer.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CoreFTP.FTPServer
+PackageVersion: 2.0-749

```

```suggestion
PackageVersion: 2.0.749
```
Semver is preferred, especially since this package doesn't write a DisplayVersion to Registry